components/atom/button/src/index.scss

```diff
@@ -0,0 +1,139 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$c-button: $c-primary !default;
+$c-button-light: color-variation($c-primary, 4) !default;
+$c-button-dark: $c-primary-dark !default;
+$c-button-light-contrast: color-variation($c-primary, 3) !default;
+$c-button-white: #ffffff !default;
+$bw-button: 1px !default;
+
+// Set all color properties for text and icons
+@mixin button-foreground-color ($color) {
+  color: $color;
+
+  & svg path {
```

Comment:
I would love to do this too but someone once told to me that we do not apply styles directly on tags and also that we do no exceptions so we don't have to memorize that exceptions. Maybe it could be reopened to discussion...

Reply:
We reference out tag with classes. But in this case, It has to be because it's not part of our markup.